CHANGELOG.md

```diff
@@ -4,6 +4,19 @@
   ## v8.10.0
 
 
+  * **gRPC instrumentation**
```

Comment:
```suggestion
  * **New gRPC instrumentation**
```
Take or leave this one!

Reply:
It reads the same both ways to me. If I see it without the "new", I might think that we've made updates to existing instrumentation, which is not desired. If I see it with the "new", I might think there have been new enhancements added to the existing instrumentation, which is not desired. I'll commit the suggestion to decisively go with one.